packages/neuron-shared/tests/load-env.test.ts

```diff
@@ -0,0 +1,46 @@
+import { loadEnv } from "../src/load-env"
+import fs from 'fs'
+
+
+jest.mock('fs')
```

Comment:
```suggestion
jest.mock('node:fs')
```

Reply:
I suggest use fs instead of node:fs for now, reasons:
1. [dotenv use 'fs'](https://github.com/search?q=repo%3Amotdotla%2Fdotenv+fs&type=code), so even if we use node:fs, we have to mock fs too. 
2. I see a lof of usages of 'fs' in the codebase, it's a little confusing to use 'fs' and 'node:fs' at the same time.

Reply:
> I suggest use fs instead of node:fs for now, reasons:
> 
> 1. [dotenv use 'fs'](https://github.com/search?q=repo%3Amotdotla%2Fdotenv+fs&type=code), so even if we use node:fs, we have to mock fs too.
> 2. I see a lof of usages of 'fs' in the codebase, it's a little confusing to use 'fs' and 'node:fs' at the same time.

It's reasonable